internal/datacoord/channel_manager.go

```diff
@@ -184,6 +184,13 @@ func (c *ChannelManager) Startup(ctx context.Context, nodes []int64) error {
 	return nil
 }
 
+// Close notifies the running checker.
+func (c *ChannelManager) Close() {
```

Comment:
->`Stop()`, as the method to start the component is named `Start()`, this should be `Stop()`

Reply:
`Close` is in sync with segmentManager